### PR TITLE
PR for SRVLOGIC-355: Add the Openshift Serverless Logic GA entry in the Serverless 1.33 release notes.

### PR DESCRIPTION
**Pull request by kaldesai**
**Affecting versions:** `serverless-docs-1.33`

**Tracking JIRA:** https://issues.redhat.com/browse/SRVLOGIC-355

**Doc previews:** 
- [OpenShift Serverless 1.33 release notes](https://78505--ocpdocs-pr.netlify.app/openshift-serverless/latest/about/serverless-release-notes.html#serverless-rn-1-33-0_serverless-release-notes)
- [Generally Available and Technology Preview features](https://78505--ocpdocs-pr.netlify.app/openshift-serverless/latest/about/serverless-release-notes.html#serverless-tech-preview-features_serverless-release-notes)

### CODE REVIEW

File: modules/serverless-rn-1-33-0.adoc
Comment (domhanak) [inline modules/serverless-rn-1-33-0.adoc:21]: This is probably depending on some other PR - `{ServerlessLogicOperatorName}.` is not resolving. 

Same for the documentation link.
Should be ok once we rebase on the other ones just mentioning so we do not forget

Comment (domhanak) [reply modules/serverless-rn-1-33-0.adoc:21]: Yea I see the comment n other PR's, apologies

Comment (kaldesai) [reply modules/serverless-rn-1-33-0.adoc:21]: Yes, sure! I will verify ll the links and attributes once we merge all the OSL PRs.

Comment (skopacz1) [inline modules/serverless-rn-1-33-0.adoc:21]: I think since the series of items in the first sentence has internal punctuation ("instructions on creating, managing, and deploying workflows") among the commas, using a semicolon here would be clearer:
```suggestion
* OpenShift Serverless Logic is now generally available (GA). This release includes an overview of OpenShift Serverless Logic; instructions on creating, managing, and deploying workflows; and guidelines for the installation and uninstallation of {ServerlessLogicOperatorName}. Additionally, it includes steps for configuring OpenAPI services and endpoints, and techniques for troubleshooting the services. For more information, see link:https://docs.openshift.com/serverless/latest/about/serverless-logic-overview.html[OpenShift Serverless Logic overview].  
```

See the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-semicolons#semicolons-between-items-in-a-series) for more details

Comment (skopacz1) [inline modules/serverless-rn-1-33-0.adoc:21]: I'm not familiar enough with serverless docs to know this - do you guys not use xrefs for internal links?

**Review by skopacz1 — COMMENTED** [review]
Just two small things, otherwise LGTM!

**Review by wmedvede — APPROVED** [review]
LGTM

Please double check the indications from @domhanak regarding the link when merged

File: modules/serverless-rn-1-33-0.adoc
Comment (kaldesai) [reply modules/serverless-rn-1-33-0.adoc]: Yes, added the xref link now.

**Comment by kaldesai** [comment]
/hold

​please hold off on merging the PR as we are waiting for the product team to give us the green signal (GA is on 8 July). Once we receive their approval, the PR will be ready to merge with everything in place. Until then you can verify the content and add LGTM if everything is fine.

**Review by skopacz1 — APPROVED** [review]
PR looks good to merge whenever you do request it to be merged (provided you request a merge after the PR that introduces the attribute is merged and you rebase this PR)

**Review by domhanak — APPROVED** [review]

**Pull request merged**
